webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingDefaults.scala

```diff
@@ -11,6 +11,8 @@ object VolumeTracingDefaults {
 
   val largestSegmentId = 0L
 
+  val unsetLargestSegmentId = -1L
```

Comment:
@fm3  would it be possible to deliver `null` instead of `-1` to the front-end? then, we could make the typescript type nullable which would give us type safety.

Reply:
Great, thank you!